.github/workflows/ci_tests_dev.yaml

```diff
@@ -116,6 +114,33 @@ jobs:
             pcre
             zlib
 
+      # Build and install latest GMT from GitHub
+      - name: Install GMT ${{ matrix.gmt_git_ref }} branch (Linux/macOS)
+        run: curl https://raw.githubusercontent.com/GenericMappingTools/gmt/master/ci/build-gmt.sh | bash
+        env:
+          GMT_GIT_REF: ${{ matrix.gmt_git_ref }}
+          GMT_INSTALL_DIR: ${{ github.workspace }}/gmt-install-dir
+        if: runner.os != 'Windows'
+
+      - name: Install GMT ${{ matrix.gmt_git_ref }} branch (Windows)
+        shell: cmd
+        run: |
+          git clone --depth=1 --single-branch --branch ${{ env.GMT_GIT_REF }} https://github.com/GenericMappingTools/gmt
+          cd gmt/
+          mkdir build
+          cd build
+          call "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Auxiliary\Build\vcvars64.bat"
+          cmake -G Ninja .. -DCMAKE_INSTALL_PREFIX=${{ env.GMT_INSTALL_DIR }} -DCMAKE_BUILD_TYPE=Release -DCMAKE_PREFIX_PATH=${{ env.MAMBA_ROOT_PREFIX }}\envs\pygmt\Library
```

Comment:
We're not using many of the build flags in https://github.com/conda-forge/gmt-feedstock/blob/8cf880960420215125ee43b1cbd24b3fc6d03291/recipe/bld.bat#L10-L28. Is that ok?

Reply:
We set the [`CMAKE_PREFIX_PATH`](https://cmake.org/cmake/help/latest/variable/CMAKE_PREFIX_PATH.html) flag to tell CMake where to find these libraries, so these flags are not necessary:
```
      -D FFTW3_ROOT=%LIBRARY_PREFIX% ^
      -D GDAL_ROOT=%LIBRARY_PREFIX% ^
      -D NETCDF_ROOT=%LIBRARY_PREFIX% ^
      -D PCRE_ROOT=%LIBRARY_PREFIX% ^
      -D ZLIB_ROOT=%LIBRARY_PREFIX% ^
      -D CURL_ROOT=%LIBRARY_PREFIX% ^
```

Parallelization support. Can be enable if necessary.
```
      -D GMT_ENABLE_OPENMP=TRUE ^
      -D GMT_USE_THREADS=TRUE ^
```
Not necessary:
```
      -D GMT_LIBDIR=lib ^
```

GSHHG and DCW data are not installed, but they can be automatically downloaded when needed.
```
      -D DCW_ROOT=%DCW_DIR% ^
      -D GSHHG_ROOT=%GSHHG_DIR% ^
      -D COPY_GSHHG=TRUE ^
      -D COPY_DCW=TRUE ^
```

Unnecessary flags for our "GMT Dev Tests": 
```
      -D GMT_INSTALL_TRADITIONAL_FOLDERNAMES=FALSE ^
      -D GMT_INSTALL_MODULE_LINKS=FALSE ^
```

Reply:
> Parallelization support. Can be enable if necessary.

Adding this would be good!

Reply:
Done in c6b1baacb601fec3de62389a05667a7bbee13a1e.